cmd/loom/loom.go

```diff
@@ -406,6 +411,15 @@ func newRunCommand() *cobra.Command {
 			if err != nil {
 				return err
 			}
+
+			if cfg.BlockStore.PruneOnStartup {
+				if err = store.PruneBlockStore(
```

Comment:
`NumBlocksToRetain` is not working properly since we always use `app.Store.Version()` as `minHeight`. With the current implementation, only the lastest block will be copied.

Reply:
Ah, condition was inverted, fixed now.